package.json

```diff
@@ -21,21 +21,27 @@
   "license": "MIT",
   "author": "Daniel Cousens",
   "files": [
-    "index.js"
+    "src"
   ],
-  "main": "index.js",
+  "main": "src/index.js",
+  "types": "src/index.d.ts",
   "repository": {
     "type": "git",
     "url": "https://github.com/cryptocoinjs/base-x.git"
   },
   "scripts": {
+    "build": "tsc -p ./tsconfig.json ; standard --fix",
```

Comment:
Why `;` and not `&&` ? 
Which folder is standard scoped ? src ? ts_src ?

Reply:
standard checks all *.js files recursively in the folder.

Reply:
Also, ; because even when tsc build fails it will still actually build the JS... so we should fix it.

Reply:
Hrmmm, don't you want it (CI etc, not sure what the setup here is) to fail if typechecking fails ?